Quadriga/src/main/java/edu/asu/spring/quadriga/dto/QuadrigaUserDTO.java

```diff
@@ -85,87 +88,74 @@
     @OneToMany(cascade = CascadeType.ALL, mappedBy = "quadrigaUserDTO")
     private List<NetworksDTO> networkDTOList;
     @OneToMany(cascade = CascadeType.ALL, mappedBy = "quadrigaUserDTO")
-    private List<NetworkAssignedDTO> networkAssignedList; 
+    private List<NetworkAssignedDTO> networkAssignedList;
     @OneToMany(cascade = CascadeType.ALL, mappedBy = "quadrigaUserDTO")
     private List<NetworkAnnotationsDTO> networkAnnotaionList;
     @OneToMany(cascade = CascadeType.ALL, mappedBy = "quadrigaUserDTO")
     private List<QuadrigaUserprofileDTO> quadrigaUserprofileDTOList;
-	@OneToMany(cascade = CascadeType.ALL, mappedBy = "quadrigaUserDTO")
+    @OneToMany(cascade = CascadeType.ALL, mappedBy = "quadrigaUserDTO")
     private List<QuadrigaUserDeniedDTO> quadrigaUserDeniedDTOList;
     @OneToMany(cascade = CascadeType.ALL, mappedBy = "quadrigaUserDTO")
     private List<QuadrigaUserRoleDTO> quadrigaUserRoleDTOList;
 
     public QuadrigaUserDTO() {
     }
 
-	public QuadrigaUserDTO(String username, String quadrigarole, String updatedby, Date updateddate, String createdby, Date createddate) {
-        this.username = username;
-        this.quadrigarole = quadrigarole;
-        this.updatedby = updatedby;
-        this.updateddate = updateddate;
-        this.createdby = createdby;
-        this.createddate = createddate;
+    @XmlTransient
```

Comment:
I don't think we need those annotations, do we? please remove if not necessary.

Reply:
We need those annotations if we are serializing/deserializing the object to xml documents.. I did not notice QuadrigaUserDTO being used to generate XML results (e.g. REST endpoints). I think its safe to remove @XmlTransient annotations.

Reply:
Yes, the annotations are used by JAXB I believe, but the DTOs should not be used in the API calls, they are Database layer classes.

Reply:
I noticed that most of the DTOs have these annotations.. Maybe we can create a separate task/ticket to look into this (since they are more related to REST services, than Social SignIn functionality)